ci/scripts/check-npm-dependencies.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable camelcase */
+/* eslint-disable no-console */
```

Comment:
please create `.eslintrc.js` file for this scripts and enable console here, because this utility scripts will use console everywhere and it's valid for them

Reply:
Do you think I should turn off camel case rule in this directory as well? Or just fix the variables to follow camel case convention? @Nodonisko @vdovhanych

Reply:
Turn it off too, we don't have to be that strict for utility scripts